.gitignore

```diff
@@ -20,3 +20,5 @@ compile_commands.json
 # cmps (https://github.com/leonmatthes/cmps)
 .cmps/
 
+# macOS
+.DS_Store
```

Comment:
note you could put this in your global gitignore at `~/.gitignore` or `~/.config/git/ignore` (depending on config) instead of per project

Reply:
Yet, some .gitignore lines are so common. Without that, there is a risk of such system artifacts being commited by inadvertence. There are temp files like those finishing by ~

Reply:
removed in last commit. If someone decide to have a more complete .gitignore, there are premade models on github